receiver/postgresqlreceiver/scraper_test.go

```diff
@@ -207,6 +207,19 @@ func TestScraperWithResourceAttributeFeatureGateSingle(t *testing.T) {
 		pmetrictest.IgnoreMetricDataPointsOrder(), pmetrictest.IgnoreStartTimestamp(), pmetrictest.IgnoreTimestamp()))
 }
 
+func TestScraperExcludeDatabase(t *testing.T) {
```

Comment:
I tried that, but I couldn't quite figure out how the golden files were generated in the first place, so I decided that it was enough to test that only two databases are called, not 3 as in the other multiple test since that's the only difference from the other tests

Reply:
Please see explanation here: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/pkg/golden#generating-an-expected-result-file